widget/embedded/src/components/ConfirmWalletsModal/WalletList.styles.ts

```diff
@@ -0,0 +1,34 @@
+import { keyframes, styled } from '@rango-dev/ui';
+
+export const LogoContainer = styled('div', {
+  position: 'relative',
+});
+
+const spin = keyframes({
```

Comment:
Please use camel case variable here.  Also maybe a more informative word like `SpinFrames` or `SpinAnimation` or something like this.